mp/game/momentum/scripts/game_sounds_momentum_weapons.txt

```diff
@@ -68,3 +68,27 @@
     "wave" "weapons/rifle/Energy_Rifle_Lower_Bass.wav"
 }
 
+"Weapon_RPG.SingleTF2"
+{
+    "channel" "CHAN_WEAPON"
+    "volume" "1.0"
+    "soundlevel" "SNDLVL_GUNFIRE"
+    "pitch" "PITCH_NORM"
+    
+    "wave" ")weapons/rocket_shoot.wav"
+}
+
+"BaseExplosionEffect.SoundTF2"
+{
+    "channel" "CHAN_WEAPON"
+    "volume" "1.0"
+    "soundlevel" "SNDLVL_95dB"
+    "pitch" "PITCH_NORM"
+
+    "rndwave"
+    {
+            "wave" ")weapons/explode2.wav"
```

Comment:
You're using 4 spaces for first indent then 8 spaces. Please use 4 spaces consistently.